packages/helix-shared-config/src/schemas/language.schema.json

```
@@ -0,0 +1,43 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe. All rights reserved.",
+    "This file is licensed to you under the Apache License, Version 2.0 (the \"License\");",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at http://www.apache.org/licenses/LICENSE-2.0",
+    "",
+    "Unless required by applicable law or agreed to in writing, software distributed under",
+    "the License is distributed on an \"AS IS\" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS",
+    "OF ANY KIND, either express or implied. See the License for the specific language",
+    "governing permissions and limitations under the License."
+  ],
+  "$id": "https://ns.adobe.com/helix/shared/language",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Language",
+  "properties": {
+    "source": {
+      "type": "string",
+      "title": "Source",
+      "description": "The source contentbus path to get records from. These records should at least contain a path property."
```

`content bus path` includes the content bus id, IMO. better use resource path:

```suggestion
      "description": "The source resource path to get records from. These records should at least contain a path property."
```